install.sh

```diff
@@ -42,6 +42,18 @@ runAsRoot() {
   $CMD
 }
 
+# scurl invokes `curl` with secure defaults
+scurl() {
+  # - `--proto =https` requires that all URLs use HTTPS. Attempts to call http://
+  #   URLs will fail.
+  # - `--tlsv1.2` ensures that at least TLS v1.2 is used, disabling less secure
+  #   prior TLS versions.
+  # - `--fail` ensures that the command fails if HTTP response is not 2xx.
+  # - `--show-error` causes curl to output error messages when it fails (when
+  #   also invoked with -s|--silent).
+  curl --proto =https --tlsv1.2 --fail --show-error "$@"
```

Comment:
```suggestion
  curl --proto "=https" --tlsv1.2 --fail --show-error "$@"
```
Quoting this as the unquoted leading `=` breaks it in zsh (at least my setup)